packages/office-ui-fabric-react/src/components/Slider/Slider.tsx

```diff
@@ -18,6 +18,9 @@ export interface ISliderState {
   renderedValue?: number;
 }
 
+/**
```

Comment:
@micahgodbolt you were the last person to touch this code.  Do you know of any reason not to remove this in v6?

Reply:
yank away